test/functional/test-custom-element.js

```diff
@@ -26,1295 +26,1299 @@ import * as lolex from 'lolex';
 
 
 describes.realWin('CustomElement', {amp: true}, env => {
-  let win, doc, ampdoc;
-  let resources;
-  let resourcesMock;
-  let clock;
-  let testElementGetInsersectionElementLayoutBox;
-  let container;
-  let ElementClass, StubElementClass;
-
-  let testElementCreatedCallback;
-  let testElementPreconnectCallback;
-  let testElementFirstAttachedCallback;
-  let testElementBuildCallback;
-  let testElementCreatePlaceholderCallback;
-  let testElementLayoutCallback;
-  let testElementFirstLayoutCompleted;
-  let testElementViewportCallback;
-  let testElementUnlayoutCallback;
-  let testElementPauseCallback;
-  let testElementResumeCallback;
-
-  class TestElement extends BaseElement {
-    isLayoutSupported(unusedLayout) {
-      return true;
-    }
-    createdCallback() {
-      testElementCreatedCallback();
-    }
-    preconnectCallback(onLayout) {
-      testElementPreconnectCallback(onLayout);
-    }
-    firstAttachedCallback() {
-      testElementFirstAttachedCallback();
-    }
-    buildCallback() {
-      testElementBuildCallback();
-    }
-    createPlaceholderCallback() {
-      testElementCreatePlaceholderCallback();
-    }
-    layoutCallback() {
-      testElementLayoutCallback();
-      return Promise.resolve();
-    }
-    firstLayoutCompleted() {
-      testElementFirstLayoutCompleted();
-    }
-    viewportCallback(inViewport) {
-      testElementViewportCallback(inViewport);
-    }
-    getIntersectionElementLayoutBox() {
-      testElementGetInsersectionElementLayoutBox();
-      return {top: 10, left: 10, width: 11, height: 1};
-    }
-    unlayoutCallback() {
-      testElementUnlayoutCallback();
-      return true;
-    }
-    pauseCallback() {
-      testElementPauseCallback();
-    }
-    resumeCallback() {
-      testElementResumeCallback();
+  // TODO(dvoytenko): Make this test work on Safari. See #11827.
+  describe.configure().skipSafari().run('CustomElement', () => {
```

Comment:
GitHub UI doesn't make it clear what actually changed here. Is there anything changes other than `skipSafari`?

Reply:
No, this just adds the `skipSafari`.

Did you click the link I sent you? https://github.com/ampproject/amphtml/pull/11827/files?w=1 Adding `?w=1` will filter out whitespace changes.

Reply:
Whoa! had no idea about `?w=1` stealing it for the next time I am assigned tip of the week.